torch/csrc/jit/pybind_utils.h

```diff
@@ -696,9 +686,8 @@ inline py::object toPyObject(IValue ivalue) {
     }
 
     auto pyCu = get_python_cu();
-    auto res = tryToConvertToCustomClass(obj);
-    if (res.has_value()) {
-      return res.value();
+    if (obj->name().find("__torch__.torch.classes") == 0) {
```

Comment:
Now that we can bind directly to py::Object should we just universally return one rather than doing the copy below?